src/renderer/components/ft-select/ft-select.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <div class="select">
     <select
+      :id="sanitizeForHtmlId(placeholder)"
```

Comment:
you could make this a computer property so the function only has to run once instead of 3 times